### PR TITLE
start to enable pylint in ci

### PR DESCRIPTION
**Pull request by guschmue**
things under tf2onnx/*.py are now find, optimizer, lstm and tests still need to be done.

### CODE REVIEW

File: tf2onnx/optimizer/transpose_optimizer.py [hunk 1]
```diff
@@ -125,8 +136,8 @@ def _update_graph_nodes(self, nodes_to_extend, nodes_to_remove, has_input_change
         if nodes_to_remove:
             for n in nodes_to_remove:
                 ops.remove(n)
-        
-        if nodes_to_extend and len(nodes_to_extend) > 0:
+
+        if nodes_to_extend and nodes_to_extend:
```
Comment (pengwa) [inline tf2onnx/optimizer/transpose_optimizer.py]: by intention here?

Comment (guschmue) [reply tf2onnx/optimizer/transpose_optimizer.py]: yes, if nodes_to_extend  is the same as len(nodes_to_extend) != 0

Comment (pengwa) [reply tf2onnx/optimizer/transpose_optimizer.py]: I mean, we should not have repeated nodes_to_extend. let me fix this in my local change.

**Comment by guschmue** [comment]
thanks!!!

**Pull request merged**